src/main/java/net/mcreator/ui/dialogs/MCItemSelectorDialog.java

```diff
@@ -190,7 +192,7 @@ public MCItemSelectorDialog(MCreator mcreator, MCItem.ListProvider blocksConsume
 
 		add("Center", mainComponent);
 
-		setSize(881, 425);
+		setSize(970, 425);
```

Comment:
Consider adding potions filter and wider screen only when potions are enabled, so users will not complain about potion button resulting in empty list for selectors where potions will not be present